src/Integration/Binding/ProjectBindingOperation.cs

```diff
@@ -28,6 +28,10 @@
 
 namespace SonarLint.VisualStudio.Integration.Binding
 {
+    // Legacy connected mode:
+    // * make binding changes to a single project i.e. writes the ruleset files
+    // and updates the project file
```

Comment:
I considered renaming any legacy-only classes by prefixing them with _Legacy_, but decided not to do it now. It might be worth doing after the new connected mode is working.